### PR TITLE
Added SMP Project

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR fixes #

**Notes for Reviewers**
The page can be viewed through projects page. I have made SMP project clickable.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Project-grid/index.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const ProjectPage = () => {
                             </Link>
                         </Col>
                         <Col xs={12} sm={4}>
-                            <Link to="#">  
+                            <Link to="/projects/service-mesh-performance">
```
Comment (Tanuj22) [inline src/sections/Project-grid/index.js:29]: Can't the projects be populated automatically as it is being done in the blog and news part?

File: src/collections/projects/smp/index.mdx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    <strong>standardize service mesh value meausurement</strong>.
+    </h4>
+  </div>
+</div>
+
+<p>The Service Mesh Performance Working Group is hosted within CNCF SIG Network. All are welcome to participate. This group is defining the Service Mesh Performance (SMP). Using SMP, MeshMark provides a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments.</p>
+<h3 class="center">SMP is a collaborative effort of Layer5, UT Austin, Google, and The Linux Foundation.</h3>
+
+<div>
+  <div style="margin: auto;">
```
Comment (Tanuj22) [inline src/collections/projects/smp/index.mdx:30]: Can you remove the inline styles in here and other places?

File: src/collections/projects/smp/index.mdx [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    <p>SMP accounts for details of:</p>
+    <ul class="bullet">
```
Comment (Tanuj22) [inline src/collections/projects/smp/index.mdx:32]: Change to `className` as it shows warning in the local environment

File: src/collections/projects/smp/index.mdx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    </ul>
+  </div>
+  <div style="margin:auto;">
+    <h2 style="text-align: center;font-weight: bold;">
+    Learn more at <a href="https://smp-spec.io">smp-spec.io</a></h2>
+  </div>
+</div>
+<p>The group is also working in collaboration with the Envoy project to create easy-to-use tooling around <a href="/projects/service-mesh-distributed-performance-management">distributed performance management</a> (distributed load generation and analysis) in context of Istio, Consul, Tanzu Service Mesh, Network Service Mesh, App Mesh, Linkerd, and so on.</p>
+
+<div class="row" style="margin: auto;">
```
Comment (Tanuj22) [inline src/collections/projects/smp/index.mdx:46]: Can't the `Row` ,`Col` and `Container` components be used?

File: src/collections/projects/smp/index.mdx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+      might reduce if our application capacity was significantly
+      larger than 100.
+    </Col>
+  </Row>
+</div>
+
+<br />
+<Row>
+  <Col className="center-col" sm={12} md={6}>
+    <blockquote className="twitter-tweet">
```
Comment (Tanuj22) [inline src/collections/projects/smp/index.mdx:129]: Can you use the React twitter component used on in the twitter

File: src/collections/projects/smp/index.mdx [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+  </h4>
+</div> 
+
+<div>
+  <h3 className="black-text">Performance of Envoy Filters</h3>
+  The following analysis compares native Envoy filter performance to WebAssembly (WASM) filter performance using Rust.
+  <br /><br />
+  <Row>
+    <Col sm={12} md={5}>
+      <a href="/assets/projects/smp/kubecon-eu/native-and-wasm-at-capacity-100rps.png">
```
Comment (Tanuj22) [inline src/collections/projects/smp/index.mdx:63]: These links should be like {img1}

File: src/collections/projects/smp/index.mdx [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+      the native rate-limiting case. Our max latency is slightly lower
+      than native, but we attribute it to various other system effects
+      like TLS handshake and network latencies that usually contribute
+      to the maximum tail latency.
+    </Col>
+  </Row>
+  <br />< br/>
+  <Row>
+    <Col sm={12} md={5}>
+    <a href="/assets/projects/smp/kubecon-eu/latency-at-scale.png">
```
Comment (Tanuj22) [inline src/collections/projects/smp/index.mdx:89]: same

File: src/collections/projects/smp/index.mdx [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+    sidecar and we start to avoid complex network redirection and
+    invocation of a new container/instance. We also notice that the
+    average latency of requests is lower than in the case of native
+    client.
+    </Col>
+  </Row>
+  <br /><br />
+  <Row>
+    <Col className="center-col" sm={12} md={3}>
+      <a href="/assets/projects/smp/kubecon-eu//client-capacity.png">
```
Comment (Tanuj22) [inline src/collections/projects/smp/index.mdx:111]: same

**Review by Tanuj22 — COMMENTED** [review]
Why am I seeing wrong image :sweat_smile: 
Screenshot from 2020-09-11 00-36-11

**Comment by Nikhil-Ladha** [comment]
Is the static data added here just for a reference of the view or will be like this? As we, can have dynamic data as we have in blogs, right?

**Comment by Tanuj22** [comment]
> Is the static data added here just for a reference of the view or will be like this? As we, can have dynamic data as we have in blogs, right?

It is just for reference. We can have dynamic data like blogs.

**Review by leecalcote — APPROVED** [review]
Good to get this content to the new site. We'll want to work on some additional custom modules for projects.

Additional content for SMP here - https://calcotestudios.com/talks/decks/slides-kubecon-eu-2020-discreetly-studying-the-effects-of-individual-traffic-control-functions.html#/5/4 

The missing image on this page is just the Envoy logo.

**Pull request merged**